test/cli/cache-dsl/cache-dsl.out

```diff
@@ -25,7 +25,7 @@ test/cli/cache-dsl/attr_accessor.rb:8: Method `prop` does not exist on `A` https
 Errors: 2
 test/cli/cache-dsl/attr_accessor.rb:7: Method `prop=` does not exist on `A` https://srb.help/7003
      7 |a.prop = 7
-        ^^^^^^^^^^
+          ^^^^^^
```

Comment:
Elsewhere in the review (I've lost track of where) I think I was thinking out loud on what the best choice is for the span given a send LHS. Wasn't thinking that carets are one of the users of that info. :) Given that, I think I  do like including the ` =` in the method loc.